components/_patterns/02-molecules/address-bar/partials/_address-bar--ctas.twig

```diff
@@ -0,0 +1,10 @@
+{% if address_bar__link__url %}
+  {% include "@atoms/01-links/link/link.twig" with {
+    link_base_class: 'cta',
+    link_modifiers: address_bar__link_modifiers|default(['secondary']),
+    link_content: address_bar__link__text,
+    link_url: address_bar__link__url,
+  } %}
+{% else %}
+  {{ address_bar__link__alternate_text }}
```

Comment:
<img width="588" alt="Screen Shot 2019-09-09 at 4 20 39 PM" src="https://user-images.githubusercontent.com/1720093/64563618-ca1e3380-d31d-11e9-81f9-39d1875f7c86.png">

I think we'll want to wrap this in something to style (like a `span` or something) as the designs in Figma have a slight padding around the "This exhibition is free..." language.

(That black line isn't part of the designs, obviously, just added it to make the spacing more obvious.)